modules/mbedtls/tests/test_crypto_mbedtls.cpp

```diff
@@ -60,4 +61,42 @@ void hmac_context_digest_test(HashingContext::HashType ht, String expected_hex)
 	String hex = String::hex_encode_buffer(digest.ptr(), digest.size());
 	CHECK(hex == expected_hex);
 }
+
+Ref<CryptoKey> create_crypto_key(String key_path, bool public_only) {
```

Comment:
```suggestion
Ref<CryptoKey> create_crypto_key(const String &p_key_path, bool p_public_only) {
```
For all

Reply:
Hi @AThousandShips 👋 

I have actioned this. Please double check though! 😄